File: PHP/uberX.php
<?php
require_once ('Car.php');
class uberX extends Car {
    public $brand;
    public $model;

    public __construct($license, $driver, $brand, $model){
        parent::__construct($license,$driver)
        $this->brand = $brand;
        $this->model = $model;
    }
    
}
?>

File: JAVA/UberX.java
package JAVA;

class UberX extends Car{
    String brand;
    String Model;

    public UberX(String license, Account driver, String brand, String Model){
        super(license, driver);
        this.brand = brand;
        this.Model = Model;
    }  

    @Override
    void printDataCar() {
        super.printDataCar();
        System.out.println("Modelo: " + Model + " Brand: " + brand);
    }
}